Gulpfile.js

```diff
@@ -29,6 +29,7 @@ const svgmin = require( 'gulp-svgmin' );
 const svgstore = require( 'gulp-svgstore' );
 const uglify = require( 'gulp-uglify' );
 const wpPot = require( 'gulp-wp-pot' );
+const replace = require( 'gulp-rename' );
```

Comment:
What does this do/why was it added? This is the only question I have around the PR – everything else is looking good! Could just use some details on why this was required. I feel like I maybe remember a conversation you and I had (maybe?), but would be good to have it noted here as well.

Reply:
https://webdevstudios.slack.com/archives/C162V538T/p1544212694089200?thread_ts=1544212070.085600&cid=C162V538T

Reply:
Ahh, that's right. I knew I had seen it somewhere but couldn't recall. Thank you!